scripts/src/primazactl/primazaworker/workercluster.py

```diff
@@ -29,23 +29,24 @@ def __init__(
             cluster_environment: str,
             tenant: str,
             internal_url: str | None = None,
+            pidentity_namespace: str | None = None,
             ):
 
         sa_name, _ = names.get_identity_names(tenant, cluster_environment)
-        super().__init__(WORKER_NAMESPACE,
-                         context,
+        super().__init__(context,
                          sa_name,
                          cluster_environment,
                          kubeconfig_file,
                          config_file,
                          cluster_environment,
                          tenant,
-                         internal_url)
+                         internal_url,
+                         pidentity_namespace)
```

Comment:
when calling super you will need pidentity_namespace to be the first parameter in place of WORKER_NAMESPACE